server.js

```diff
@@ -0,0 +1,41 @@
+var cluster = require('cluster');
+var path = require('path');
+var log = require('util').debuglog('master');
+
+if(cluster.isMaster) {
+	cluster.setupMaster();
+
+	log('Starting workers');
+	// start the api server
+	cluster.fork({
+		script: 'src/api/index.js',
+		port: process.env.npm_package_config_api_port
+	});
+	log('Started api server');
+
+	// start the account zone server
+	cluster.fork({
+		script: 'src/account/index.js',
+		port: process.env.npm_package_config_account_port,
+		static_root: process.env.npm_package_config_account_root
+	});
+	log('Started account server');
+
+	// start the admin zone server
+	cluster.fork({
+		script: 'src/admin/index.js',
+		port: process.env.npm_package_config_admin_port,
+		static_root: process.env.npm_package_config_admin_root
+	});
+	log('Started account server');
+
+	process.on('SIGINT', function () {
+		// cleanly exit workers
+		log('Caught signal, killing workers');
+		for(var id in cluster.workers) {
```

Comment:
`foreach` (which is actually `for(... of ...)`) in javascript would require the `--harmony` flag.